.github/workflows/ci.yml

```diff
@@ -70,69 +118,59 @@ jobs:
     #
     # ¹ https://github.com/nextstrain/nextstrain.org/settings/environments
     environment:
-      name: heroku
-      url: https://next.nextstrain.org
+      name: ${{ inputs.heroku-app || 'nextstrain-canary' }}
+      url: ${{ inputs.heroku-app && format('https://{0}.herokuapp.com', inputs.heroku-app) || 'https://next.nextstrain.org' }}
```

Comment:
On the [deployments](https://github.com/nextstrain/nextstrain.org/deployments) page, `nextstrain-canary` maps to `next.nextstrain.org` and `nextstrain-dev` to `nextstrain-dev.herokuapp.com` (as opposed to `dev.nextstrain.org`). I thought that was weird, but it makes sense after looking back at this.

Reply:
Yeah, we could improve that various ways later.